src/main/java/christmas/domain/EventType.java

```diff
@@ -0,0 +1,60 @@
+package christmas.domain;
+
+import christmas.service.event.EventDayCalculator;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+public enum EventType {
+    CHRISTMAS("크리스마스 디데이 할인", EventDayCalculator.christmasDay(), orderDay -> {
+        final int basicDiscountAmount = 1000;
+        final int discountAmountUnit = 100;
+        return (basicDiscountAmount + discountAmountUnit * (orderDay - 1)) * -1;
+    }),
+    WEEKDAY("평일 할인", EventDayCalculator.weekDay(), dessertCount -> {
+        final int discountAmountUnit = 2023;
+        return (discountAmountUnit * dessertCount) * -1;
+    }),
+    WEEKEND("주말 할인", EventDayCalculator.weekEnd(), mainCourseCount -> {
+        final int discountAmountUnit = 2023;
+        return (discountAmountUnit * mainCourseCount) * -1;
+    }),
+    SPECIAL("특별 할인", EventDayCalculator.specialDay(), anything -> {
+        return -1000;
+    });
```

Comment:
람다식이 딱 한 줄의 return문만을 가질 경우 return을 포함한 람다 바디를 리턴 값만으로 단축시킬 수 있습니다!
```suggestion
    SPECIAL("특별 할인", EventDayCalculator.specialDay(), anything ->  -1000);
```